src/main/java/nl/ordina/jobcrawler/controller/VacancyScraper.java

```diff
@@ -38,14 +39,14 @@ public String getSEARCH_URL() {
          */
         List<Vacancy> vacancies = new ArrayList<>();
         List<VacancyURLs> vacancyURLs = getVacancyURLs();
-        for(VacancyURLs vacancyURL : vacancyURLs){
+        for (VacancyURLs vacancyURL : vacancyURLs) {
             Document doc = connectionDocumentService.getConnection(vacancyURL.getUrl());
-            if(doc != null) {
+            if (doc != null) {
                 Vacancy vacancy = Vacancy.builder()
                         .vacancyURL(vacancyURL.getUrl())
                         .broker(BROKER)
                         .hours(vacancyURL.getHours())
-                        .skillSet(new ArrayList<>())
+                        .skills(new HashSet<>())
```

Comment:
Why is it now a hashset?

Reply:
before the many-to-many relationship skills was a list of strings. Now it is a Set; a Set is initialized with a HashSet.